### ISSUE TITLE
Import normalize.css from npm?

### ISSUE THREAD

**Issue by tymondesigns** [inuitcss/inuitcss#128]
Hey chaps, just caught up with all the recent changes, awesome work!

After noticing the inclusion of `sass-mq` externally, I wondered if this approach should be applied to necolas' `normalize.css`

Not sure if this has been discussed before, so apologies if so.

I'd be happy to submit a PR if this is a good shout ?

```sass
// edit - this works, although a bit wordy
@import "node_modules/normalize.css/normalize";
```

**Comment by csswizardry** [comment]
I’d love to! The issue lies in `@import`ing a `.css` file into a Sass project though. Ideas? :)

**Comment by tymondesigns** [comment]
I'm not sure about ruby-sass but it definitely works with node-sass using the code above.

**Comment by tymondesigns** [comment]
in-fact here is the libsass PR for reference https://github.com/sass/libsass/pull/754

**Comment by tymondesigns** [comment]
after some more investigation looks like sass/sass has still not added this feature out of the box https://github.com/sass/sass/issues/193. Apparently it's promised in v4.

I guess that's pretty much a show stopper given that ruby-sass support is a must!

I guess the only other option I can think of is to use one of the normalize sass forks e.g. [this](https://www.npmjs.com/package/node-normalize-scss)

I can understand why that might not be desirable though :/

**Comment by anselmh** [comment]
Yes, if there's support for CSS partial import we can certainly use the original dependency.
If we use our own or another fork doesn't matter though.

**Comment by csshugs** [comment]
Unfortunately, [this](https://github.com/ranjandatta/node-normalize-scss) fork is not provided as a bower component. I assume, we don't want to dictate either npm or bower to our users. So I propose providing a dedicated `inuitcss-normalize.css` fork, with a `package.json` *and* a `bower.json`.

**Comment by nenadjelovac** [comment]
>So I propose providing a dedicated inuitcss-normalize.css fork, with a package.json and a bower.json

Why would we then split it from the main repo?

**Comment by csshugs** [comment]
@nenadjelovac Good question! I don't know 😆 

Maybe there are reasons. Anyone know some reasons?
Otherwise, ignore my suggestion. We then should just stick to the solution we already use.

**Comment by tymondesigns** [comment]
My reasons for suggesting this were to remove the debt of having to maintain a fork, so moving it out into a separate repo doesn't solve that

**Comment by csswizardry** [comment]
@tymondesigns: Yeah, I’m totally on board with your idea. As soon as Sass can `@import` CSS files, we”ll get rid of our fork and use an external dependency.

Am I okay to close…?

**Comment by tymondesigns** [comment]
Yea go ahead 👍 

We can only wait... :)

**Comment by csswizardry** [comment]
@tymondesigns: Thank you. I really appreciate your input :)

**Comment by tymondesigns** [comment]
@csswizardry No problem! Happy to help wherever I can
